src/main/java/com/thevoxelbox/voxelsniper/brush/type/performer/CylinderBrush.java

```diff
@@ -22,7 +25,7 @@ public void handleCommand(String[] parameters, Snipe snipe) {
             }
             if (parameter.equalsIgnoreCase("info")) {
                 messenger.sendMessage(ChatColor.GOLD + "Cylinder Brush Parameters:");
-                messenger.sendMessage(ChatColor.AQUA + "/b c h[number] -- set the cylinder v.voxelHeight.  Default is 1.");
+                messenger.sendMessage(ChatColor.AQUA + "/b c h[number] -- set the cylinder v.voxelHeight. Default is 1.");
                 messenger.sendMessage(ChatColor.DARK_AQUA + "/b c true -- will use a true circle algorithm instead of the skinnier version with classic sniper nubs. /b b false will switch back. (false is default)");
                 messenger.sendMessage(ChatColor.DARK_BLUE + "/b c c[number] -- set the origin of the cylinder compared to the target block. Positive numbers will move the cylinder upward, negative will move it downward.");
```

Comment:
See FlatOceanBrush comment & dark blue is a horrible color to display anything ingame. Putting this here for the entire project, replace unreadable colors like black or dark blue in spotted.